sql/plan/show_create_table_test.go

```diff
@@ -40,7 +41,7 @@ func TestShowCreateTable(t *testing.T) {
 	expected := sql.NewRow(
 		table.Name(),
 		"CREATE TABLE `test-table` (\n  `baz` text NOT NULL,\n"+
-			"  `zab` integer DEFAULT 0,\n"+
+			"  `zab` int DEFAULT 0,\n"+
```

Comment:
You should add a TODO to change the casing behavior here. It's dumb that we lowercase these names when reserved words are upper case in every other context

Reply:
I just fixed it instead, they were calling `ToLower` so I just removed it.